pkg/util/aad/aad.go

```diff
@@ -125,3 +125,19 @@ func authenticateServicePrincipalToken(ctx context.Context, log *logrus.Entry, a
 
 	return nil
 }
+
+// GetObjectId extracts the "oid" claim from a given access jwtToken
+func GetObjectId(jwtToken string) (string, error) {
+	p := jwt.NewParser(jwt.WithoutClaimsValidation())
+	c := &custom{}
+	_, _, err := p.ParseUnverified(jwtToken, c)
```

Comment:
The JWT Go package [recommends](https://pkg.go.dev/github.com/golang-jwt/jwt/v4#Parser.ParseUnverified) using `Parse` because `ParseUnverified` doesn't validate signatures. Is there a reason we are using `ParseUnverified` instead of `Parse`?

Reply:
I thought about this and my conclusion was this is for getting the oid claim rather than doing an authentication (there's just not an easy way to get it from aad).  It also does this immediately after calling AAD.  The sender and the receiver are the same party (in this case the RP), unlike in other scenarios where the sender and receiver can't be trusted. 

ParseUnverified is also used in this codebase to get the tenant ID and roles as well.

Reply:
Makes sense. Maybe we when deprecate go-autorest, we can revisit this and see if it makes sense to verify the token.